docs/content/2.configuration/2.nuxt-config.md

```diff
@@ -61,7 +61,7 @@ The `AUTH_ORIGIN` environment variable takes precendence over the `origin` confi
 The origin must be set so that `nuxt-auth` can ensure that callbacks for authentication are correct. The `origin` consists out of (up to) 3 parts:
 - scheme: `http` or `https`
 - host: e.g., `localhost`, `example.org`, `www.sidebase.io`
-- port: e.g., `:3000`, `:4444`; leave empty to implicitly set `:80` (this is an internet convention, don't ask)
+- port: e.g., `:3000`, `:4444`; leave empty to implicitly set `:80` for http, and `:443` for https (this is an internet convention, don't ask)
```

Comment:
Can you please also add this to the "future" docs version in the `v0.6` folder?

Reply:
on it.